services/azure/Compute/virtualMachines.yml

```diff
@@ -40,45 +47,79 @@ privileges:
     notes: >-
       Stops and releases allocation; VMs become unavailable until started.
   runCommand:
-    risks: [exfiltration:data, impact:hijack]
+    risks:
+      [
+        exfiltration:data,
+        exfiltration:crypto,
+        impact:hijack,
+        persistence:account,
+      ]
     notes: >-
       Executes arbitrary commands inside the guest OS, enabling data theft or tampering.
   login:
-    risks: [exfiltration:data, impact:hijack]
+    risks:
+      - collection:data
+      - destruction:data
+      - discovery:network
+      - discovery:policy
+      - escalation:lateral
+      - exfiltration:data
+      - impact:defacement
+      - impact:hijack
     notes: >-
       Allows interactive user login to the VM using Azure AD-backed authentication
       (Microsoft.Compute/virtualMachines/login/action). Grants OS user access.
   loginAsAdmin:
-    risks: [exfiltration:data, impact:hijack, escalation:privilege]
+    risks:
+      - collection:data
+      - destruction:data
+      - discovery:network
+      - discovery:policy
+      - escalation:lateral
+      - exfiltration:data
+      - impact:defacement
+      - impact:hijack
     notes: >-
       Allows interactive administrator login to the VM using Azure AD-backed authentication
       (Microsoft.Compute/virtualMachines/loginAsAdmin/action). Grants privileged OS access.
   extensionsWrite:
-    risks: [exfiltration:data, escalation:privilege]
+    risks:
+      [
+        exfiltration:data,
+        escalation:privilege,
+        impact:hijack,
+        persistence:account,
+      ]
     notes: >-
       Installing or updating VM extensions can run privileged code.
   extensionsDelete:
-    risks: [impact:dos, destruction:infra]
+    risks: [impact:dos, destruction:infra, destruction:defense]
     notes: >-
       Removing required extensions (e.g., monitoring/agents) can break operations or visibility.
   attachManagedDisk:
-    risks: [exfiltration:data, destruction:data]
+    risks:
+      [
+        exfiltration:data,
+        exfiltration:crypto,
```

Comment:
I don't follow `exfiltration:crypto`.

The link we have in the catalog for that points to [Unsecured Credentials](https://attack.mitre.org/techniques/T1552/) which says:

>Adversaries may search compromised systems to find and obtain insecurely stored credentials.

I guess they could save stuff to the disk? We also classify operations that simply list things as `exfiltration`, so that's what's confusing me.  Maybe those should just be `collection` instead.